Cargo.toml

```diff
@@ -9,3 +9,4 @@ edition = "2018"
 [dependencies]
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
+ureq = "1.5.1"
```

Comment:
I didn't know ureq!
I believe blocking is fine for now. 
What was wrong with reqwest?

Reply:
I didn't find how to use it without async... 🤷‍♂️

Reply:
reqwest::blocking::get? 

There is an example in the README: https://github.com/seanmonstar/reqwest

Is it not working as expected?

Reply:
Switched back to `reqwest`